internal/oci/stats.go

```diff
@@ -24,24 +24,27 @@ type ContainerStats struct {
 }
 
 // Returns the total number of bytes transmitted and received for the given container stats
-func getContainerNetIO(stats *libcontainer.Stats) (received, transmitted uint64) {
-	for _, iface := range stats.Interfaces {
-		received += iface.RxBytes
-		transmitted += iface.TxBytes
-	}
-	return
+func getContainerNetIO() (received, transmitted uint64) {
+	/*
+		TODO(saschagrunert): find a way to retrieve the data here
```

Comment:
This triggers more effort on our side and will change as well when migrating to libpod.